### DISCUSSION TITLE
ERROR FLUTTER

### DISCUSSION THREAD

**Discussion by yanquirm** [supabase/supabase#4943] [Questions] [answered]
Exception has occurred.
SocketException (SocketException: Failed host lookup: 'yngcmjmnbtaqryonamac.supabase.co' (OS Error: nodename nor servname provided, or not known, errno = 8))

**Comment by inian** [comment, marked as answer]
@yanquirm I think this issue has been fixed, but if you are still experiencing an error, create a support ticket at https://app.supabase.io/support/new
